### ISSUE TITLE
LightBox

### ISSUE THREAD

**Issue by zmoddynamics** [2amigos/yii2-gallery-widget#14]
Hi, thanks for the great extension. I have a question about a specific type of implementation. I'm looking to have a video button "overlay" on all the images in the carousel. I want the video to open in a lightbox. Is this possible?

**Comment by tonydspaniard** [comment]
Yes is possible, in fact it does it by default. Check the plugin source page for options. http://blueimp.github.io/Gallery/

**Comment by patel28** [comment]
Is this possible to enable rotation with this widget ???
